### ISSUE TITLE
There’s no “first” or “last” links in the pagination for the files listing

### ISSUE THREAD

**Issue by atopal** [web3-storage/web3.storage#434]
On the Files page, there is no way to get back to the first page after paging through other than going through the main nav. We should add "First" and "Last" links in addition to the current "previous" and "next" links.

**Comment by dchoi27** [comment]
Can get started on this but "Last" is blocked on the DB migration

**Comment by atopal** [comment]
Yeah, that's fine. Last is probably less useful than First anyway.
